lib/email_reply_parser.rb

```diff
@@ -134,6 +138,33 @@ def read(text)
     EMPTY = "".freeze
     SIGNATURE = '(?m)(--\s*$|__\s*$|\w-$)|(^(\w+\s+){1,3}ym morf tneS$)'
```

Comment:
:speech_balloon: No multilanguage for signature detection?

Reply:
Not yet, we can keep improving it in the future